products/experiments/backend/max_tools.py

```diff
@@ -1,154 +1,15 @@
-from typing import Any, Literal
+from typing import Any
 
 from posthoganalytics import capture_exception
 from pydantic import BaseModel, Field
 
 from posthog.schema import MaxExperimentSummaryContext
 
-from posthog.models import Experiment, FeatureFlag
-from posthog.sync import database_sync_to_async
-
 from ee.hogai.llm import MaxChatOpenAI
 from ee.hogai.tool import MaxTool
 
 from .prompts import EXPERIMENT_SUMMARY_BAYESIAN_PROMPT, EXPERIMENT_SUMMARY_FREQUENTIST_PROMPT
 
-
-class CreateExperimentArgs(BaseModel):
-    name: str = Field(description="Experiment name - should clearly describe what is being tested")
-    feature_flag_key: str = Field(
-        description="Feature flag key (letters, numbers, hyphens, underscores only). Will create a new flag if it doesn't exist."
-    )
-    description: str | None = Field(
-        default=None,
-        description="Detailed description of the experiment hypothesis, what changes are being tested, and expected outcomes",
-    )
-    type: Literal["product", "web"] = Field(
-        default="product",
-        description="Experiment type: 'product' for backend/API changes, 'web' for frontend UI changes",
-    )
-
-
-class CreateExperimentTool(MaxTool):
```

Comment:
These tools should still live in the product folders IMO, as the AI tools reflect the relevant product analogously to its REST API, also destined to leave here.
I.e. the product teams maintain the "dumb" product tools, while PostHog Ai's `ee/hogai` maintains the orchestration of those tools.